elasticdl/python/tests/embedding_table_test.py

```diff
@@ -0,0 +1,69 @@
+import operator
+import unittest
+from functools import reduce
+
+import numpy as np
+
+from elasticdl.proto.elasticdl_pb2 import EmbeddingTableInfo
+from elasticdl.python.pserver.embedding_table import (
+    EmbeddingTable,
+    create_embedding_table,
+)
+
+
+def prod(iterable):
```

Comment:
No need to define a method, just use `np.prod(iterable)`.

Reply:
Done